libs/category/src/drivers/magento/transformers/custom-metadata-attributes-transformer.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
```

Comment:
This doesn't need to be a class, it can just be a set of pure functions. I think it should be split apart as such.

Reply:
done